pyvista/core/filters/data_set.py

```diff
@@ -1977,7 +1964,7 @@ def glyph(
                 alg.SetIndexModeToOff()
 
         if isinstance(scale, str):
-            dataset.set_active_scalars(scale, 'cell')
+            dataset.set_active_scalars(scale, 'point')
```

Comment:
I'm fine with this, but I personally don't think it is worth the breaking change.  This is particularly painful when using OpenFOAM data that has duplicate names for arrays in point and cell data.  This change will require changes in user code in this case.

Reply:
I agree that it's not worth it to break the existing behavior. I changed this back so it prefers 'cell'.  FWIW, `dataset.set_active_scalars()` prefers `cell` by default, whereas `dataset.set_active_vectors()` prefers `point`.

Reply:
> FWIW, `dataset.set_active_scalars()` prefers `cell` by default, whereas `dataset.set_active_vectors()` prefers `point`.

Ugh.  In that case, I'm okay either way.

Reply:
There are 15 instances of `preference='point'` (or with double quotes) and 10 instances of `preference='cell'` (or with double quotes). This excludes docstring cameos, but includes code inside method bodies (so these numbers don't reflect the respective default arguments)...

Reply:
I've opened #2515 so this discussion can move out of this PR (other than discussing how to implement for this filter)